gripper/firmware/interfaces_grip_motor.cpp

```diff
@@ -31,10 +34,34 @@ struct motor_hardware::BrushedHardwareConfig brushed_motor_pins {
             .port = GPIOC,
             .pin = GPIO_PIN_11,
             .active_setting = GPIO_PIN_SET},
-    .limit_switch = {  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-cstyle-cast)
-        .port = GPIOC,
-        .pin = GPIO_PIN_2,
-        .active_setting = GPIO_PIN_SET},
+    .limit_switch =
+        {  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-cstyle-cast)
+            .port = GPIOC,
+            .pin = GPIO_PIN_2,
+            .active_setting = GPIO_PIN_SET},
+    .encoder_interrupt_freq =
+        double(GRIPPER_JAW_PWM_FREQ_HZ) / double(GRIPPER_JAW_PWM_WIDTH),
+
+    /* the expected behavior with these pid values is that the motor runs at
+     * full power until it's about 2mm away and then it slows down on that
+     * approach. final values as the error delta approachs 0.01mm is ~7
+     * which is the floor of pwm values that still move the gripper
+     * this 7 is from the windup limits, but over very short moves( <
```

Comment:
Is this an incomplete thought?

Reply:
definitely want to fix this before merging